Update SAS for Round 3 (#4412)
ppy/osu-wiki: wiki/Tournaments/SAS/1/en.md

| #3,931 | ![][flag_US] [Ninty](https://osu.ppy.sh/users/11631179) |
| #3,938 | ![][flag_BR] [AlgocomB](https://osu.ppy.sh/users/12298887) |
| #3,975 | ![][flag_NZ] [- Abby -](https://osu.ppy.sh/users/7740118) |
| #4,024 | ![][flag_ES] [game\_maker26](https://osu.ppy.sh/users/4876699) |
| #4,106 | ![][flag_CL] [-JustLenny](https://osu.ppy.sh/users/12793583) |
| #4,215 | ![][flag_KR] [S ILY](https://osu.ppy.sh/users/9766317) |
| #4,323 | ![][flag_US] [\[LS\]Crazy](https://osu.ppy.sh/users/12211556) |
| #4,503 | ![][flag_PL] [Mr\_adamello](https://osu.ppy.sh/users/7420894) |
| #4,870 | ![][flag_AR] [boligoma](https://osu.ppy.sh/users/13310563) |
| #5,017 | ![][flag_US] [walnutt](https://osu.ppy.sh/users/8617859) |
| #5,331 | ![][flag_MX] [\[LS\] Shuri](https://osu.ppy.sh/users/14593518) |
| #5,546 | ![][flag_NO] [prebert123](https://osu.ppy.sh/users/7859738) |
| #5,607 | ![][flag_TW] [Joylintp](https://osu.ppy.sh/users/13579078) |
| #5,732 | ![][flag_US] [FlynnMac](https://osu.ppy.sh/users/13857986) |
| #6,162 | ![][flag_AU] [\[LG\]aurusei](https://osu.ppy.sh/users/14430622) |
| #6,748 | ![][flag_BR] [Halo-](https://osu.ppy.sh/users/7572142) |

> *Note: Global rank number was taken in the end of registration.*

Reviewer: ```md
*Note: Global ranks were recorded at the end of registration.*
```
and move it to before the table

## Mappools

### Round 2

**[Download the mappack here! (27 MB)](https://cdn.discordapp.com/attachments/711778005399109687/772567150929641502/SAS_Round_2.zip)**

- Hybrid

>   - [plusol - Angel Temple (BringoBrango) \[Ascension (SAS edit)\]](https://osu.ppy.sh/beatmapsets/1286284#mania/2670767)

Reviewer: might as well not be a second list if it's only one map. id do something like `- Hybrid: [plusol - ...` for each of these

- Tech
  - [Kola Kid - spaceman (cut) (Vortex-) \[I always wanted you to go, into space man (1.1x)\]](https://osu.ppy.sh/beatmapsets/1290983#mania/2679801)
- LN
  - [Camellia - potential curve (Raveille) \[diatomic\]](https://osu.ppy.sh/beatmapsets/1196957#mania/2493350)
- Wildcard
  - [Equinox7 - valerian coma (-Lethal-) \[atrophy\]](https://osu.ppy.sh/beatmapsets/1286278#mania/2670752)

### Round 1

**[Download the mappack here! (20 MB)](https://cdn.discordapp.com/attachments/711916285746872412/770020679361232896/SAS_Round_1.zip)**

- Hybrid
  - [WEDNESDAY CAMPANELLA - Shakushain (Penguinosity) \[Shakushaku\]](https://osu.ppy.sh/beatmapsets/1282584#mania/2663884)
- Tech
  - [Ben Briggs - Fearless Living (RuleBlazing) \[Coward\]](https://osu.ppy.sh/beatmapsets/1283930#mania/2666220)
- LN
  - [Drop x Hatsuki Yura - Le Bijoux (HowToPlayLN) \[Jewelry\]](https://osu.ppy.sh/beatmapsets/1282938#mania/2664488)
- Wildcard
  - [Jakub Zytecki - Letters (Cokiiplay) \[Voyage\]](https://osu.ppy.sh/beatmapsets/1282382#mania/2663564)

## Match results

> **All scores listed will be the sum of all scores from each maps in respective mappool, for more detailed score from each map visit the statistics sheet available in the [links](#links) section.**

Reviewer: ```suggestion
*Note: All scores listed are the sum of scores from each mappool. More detailed results are available on the [statistics sheet](https://docs.google.com/spreadsheets/d/1SN7zflqtF9mb37IeIdKzNbeZuiBttqUL-sdU0Eexwwg/edit#gid=0).*
```

### Round 1

> October 26, 2020 - October 30, 2020:
> 

Reviewer: ```suggestion
```
seems unnecessary with the schedule at the top

| #118 | ![][flag_KR] [S ILY](https://osu.ppy.sh/users/9766317) | 0 |
| #119 | ![][flag_CO] [sebasthefast](https://osu.ppy.sh/users/14395814) | 0 |
| #120 | ![][flag_AU] [siberbule](https://osu.ppy.sh/users/11495339) | 0 |
| #121 | ![][flag_BR] [Soore](https://osu.ppy.sh/users/15753462) | 0 |
| #122 | ![][flag_ID] [Soraru Aoki](https://osu.ppy.sh/users/15636932) | 0 |
| #123 | ![][flag_US] [stupud man](https://osu.ppy.sh/users/2141612) | 0 |
| #124 | ![][flag_ID] [Suichan203](https://osu.ppy.sh/users/9821346) | 0 |
| #125 | ![][flag_PL] [szuncoll](https://osu.ppy.sh/users/12393014) | 0 |
| #126 | ![][flag_US] [Trainer-](https://osu.ppy.sh/users/9331760) | 0 |
| #127 | ![][flag_US] [treeman9923](https://osu.ppy.sh/users/14677848) | 0 |
| #128 | ![][flag_SE] [Trymatic](https://osu.ppy.sh/users/14762260) | 0 |
| #129 | ![][flag_US] [Vegan](https://osu.ppy.sh/users/9727747) | 0 |
| #130 | ![][flag_AU] [Water is good](https://osu.ppy.sh/users/8222416) | 0 |
| #131 | ![][flag_GB] [welshevo79](https://osu.ppy.sh/users/10510684) | 0 |
| #132 | ![][flag_CL] [xXCris13Xx](https://osu.ppy.sh/users/11480970) | 0 |
| #133 | ![][flag_ID] [Yangga](https://osu.ppy.sh/users/12143152) | 0 |

> - Only the top 55 player can proceed to the next round

Reviewer: ```suggestion
- Only the top 55 players proceed to the next round
```

> - The higher the score the better

Reviewer: ```suggestion
```
implied from how it's sorted i think

4. Failing to complete ruleset #2 or #3 **may result in your registration not being counted**.
5. The scoring system used will be **ScoreV2**.
6. There will be five rounds in the tournament. Each round, participants will be given 5 days (1 work week) to get the highest possible cumulative score across all 4 maps for the running round.

> 7. Maps will be played under FreeMod. Allowed mods are only the visual mods and Sudden Death mod
>    - The visual mods are consisted of: Hidden, Fade In, and Flashlight

Reviewer: ```suggestion
7. Maps will be played with FreeMod. The only allowed mods are FadeIn, Flashlight, Hidden, and SuddenDeath.
```
